osu.Game/OsuGame.cs

```diff
@@ -616,6 +620,7 @@ protected override void LoadComplete()
             loadComponentSingleFile(CreateUpdateManager(), Add, true);
 
             // overlay elements
+            loadComponentSingleFile(new ManageCollectionsDialog(), overlayContent.Add, true);
```

Comment:
Is there a reason this is a global thing? Seems like something we wouldn't want global since it's only used in one place.

Reply:
Hmm... The reasoning is the dim on `OsuFocusedOverlayContainer` doesn't behave nicely if it's placed anywhere else, I suppose I can add a local backgrop to the dialog, or maybe not dim, if you'd rather that?

Reply:
Did you try using `DialogOverlay.Push`?

Reply:
That requires it to be a `PopupDialog`, which is structurally/visually completely different and isn't (at this point) geared towards having arbitrary content placed inside it. That was the direction I was trying for originally but I thought the complexity of doing so wasn't worth it, however now that the hierarchy is simplified a bit I can give it a shot.

Reply:
Probably not going to work well since it shows a delete dialog and i don't think multiple overlapping dialogs are supported.

I'm not sure I see the issue with dimming, i'll try local so i can understand it myself.

Reply:
The issue is that the dim is applied to the game's content, which means the dialog would also be dimmed itself if placed inside SongSelect.

Reply:
i see, unfortunate. would really like to reduce this load-on-startup components especially in cases they may never be used.

let's leave it for now though, that can be a future optimisation. the only weird part about doing it this way is the rearrangeable list animates only on first display.